syft/file/classifier.go

```diff
@@ -111,11 +122,15 @@ func (c Classifier) Classify(resolver source.FileResolver, location source.Locat
 			continue
 		}
 
+		// TODO: it looks like we're getting two duplicate classificationns in some cases where
+		// VirtualPath differs from RealPath. The classifications don't show this difference
+		// Should we update the classifications to identify the virtual path that linked to the realpath?
```

Comment:
I'll remove this `TODO` if we like the VirtualPath change. It's also a reminder for me to update the schema if we go this route.